docs/source/ide-integration.rst

```diff
@@ -115,8 +115,8 @@ Output should be something similar to this:
 
 .. code:: bash 
 
-   /PATH_TO_JDK/jdk1.8.0_131/bin/java
```

Comment:
`Lines 69-71` the `right click > create tornadovm [package]` does not to exist any more. Can we please update also those instructions?